spec/services/export/report_spec.rb

```diff
@@ -318,6 +318,34 @@
     end
   end
 
+  describe "@activity_attributes" do
```

Comment:
Do we normally have a `describe` block for instance variables? The others don't seem to have them

Reply:
We don't usually, but I thought it was worth testing the headers were different depending on the type, and the existing header tests weren't fund-specific - I thought this was the easiest way to assert that the right thing was being used.

Would you rather we had specific tests for the headers themselves (e.g. update the `#headers` method's tests)? I did initially head down that route but it became a bit of an unwieldy spec to update and I didn't want to spend too long refactoring the specs to bend them into that shape when it felt like a simple conditional `if x else y` to test here!

Reply:
No strong opinions/preference - just struck me as different to the usual pattern!

Reply:
I'll leave it as-is for now, as I don't want to block this, but if a better way of testing does come to mind, please let me know / add a card, as we can definitely revisit this in future